engine/src/test/java/io/camunda/zeebe/process/test/engine/db/BytesTest.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+
+package io.camunda.zeebe.process.test.engine.db;
+
+import io.camunda.zeebe.db.impl.ZeebeDbConstants;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import org.agrona.ExpandableArrayBuffer;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class BytesTest {
+
+  @Test
+  public void compareDifferentSizeSmaller() {
```

Comment:
🔧 Most of the test cases we write use the `shouldXYZ` naming convention. Should we use it here and describe what the test should ensure?
```suggestion
  public void shouldYieldShorterByteArrayAsSmaller() {
```

Reply:
Yes, absolutely. Will make this change now.